p4c_bm/gen_json.py

```diff
@@ -1231,9 +1272,12 @@ def json_dict_create(hlir, path_field_aliases=None, p4_v1_1=False):
     dump_field_lists(json_dict, hlir)
     dump_counters(json_dict, hlir)
     dump_registers(json_dict, hlir)
-
     dump_force_arith(json_dict, hlir)
 
+    if p4_v1_1 and hlir.p4_extern_instances:  # pragma: no cover
```

Comment:
You can remove `# pragma: no cover` here; it is actually covered by your new test program.